src/aspire/storage/starfile.py

```diff
@@ -16,24 +14,24 @@ class StarFileError(Exception):
 class StarFile:
     def __init__(self, filepath="", blocks=None):
         """
-        Initialize either from a path to a STAR file or from an OrderedDict of dataframes
+        Initialize either from a path to a STAR file or from an OrderedDict of ndarrays
         """
         # if blocks are given, set self.blocks, otherwise initialize an empty OrderedDict()
+        if blocks is not None:
```

Comment:
Comment briefly on what this is supposed to do? Some sort of normalization of block data?

Reply:
# The following code repacks provided blocks as strings while handling both singleton and iterable inputs.